src/components/common/Picker/styles.js

```diff
@@ -0,0 +1,46 @@
+import { StyleSheet } from 'react-native';
+
+const baseLabel = {
+  height: 15,
+  textAlign: 'center',
+  fontWeight: 'bold'
+};
+
+const basePicker = {
+  width: 230,
+  height: 40,
```

Comment:
In other cases try to avoid on width and height fixed values. You could use percentage or auto to adjust what you want.